cmd/bee/cmd/start.go

```diff
@@ -56,15 +56,15 @@ func (c *command) initStartCmd() (err error) {
 			case "0", "silent":
 				logger = logging.New(ioutil.Discard, 0)
 			case "1", "error":
-				logger = logging.New(cmd.OutOrStdout(), logrus.ErrorLevel)
+				logger = logging.New(cmd.OutOrStderr(), logrus.ErrorLevel)
```

Comment:
why? i personally prefer to stdout, errors are anyway written to stderr afaik

Reply:
that's perfectly normal operation, to want the result of cli cmd and the log chatter kept separate.

Reply:
Yes, log messages should be sent to stderr, as usual practice, as to have the command output to stdout as the result of the execution. I have set stdout here as start does not have the resulted output, only logs, but I have nothing against using stderr for log messages.

I would also like to hear from @svetomir or @vandot if this is ok from the infrastructure point of view.

Reply:
all messages? error I understand (and i think the logger should do this regardless), but everything else? i am used that `WARN` and `ERROR` go to `stderr`, but the rest to `stdout` no?

Reply:
From a cli user point of view I would absolutely _loathe_ that, and I dare ever make the assumption that that's not the way it usually works.

Reply:
This is true for applications that produce some output. In this case, `bee start` does not produce any output and it is a long running process, with logs that can be considered as the output. Maybe even this PR is not the place to change this. This change in `bee start` should be in a separate PR.